tools/dev_console/static/styles.css

```
@@ -1,783 +1,6 @@
-  .nav-workspace,
-  .dynamic-workspace,
-  .cleanup-workspace,
-  .graph-workspace {
-    grid-template-columns: 1fr;
-  }
-
-  .graph-controls {
-    grid-template-columns: 1fr;
-  }
-}
+@import url('./styles/tokens.css');
```

Stylelint (17.12.0)

[error] Expected "url('./styles/tokens.css')" to be "'./styles/tokens.css'" (import-notation)

(import-notation)